benchmarks/dynamo/pr_time_benchmarks/expected_results.csv

```diff
@@ -1,66 +1,65 @@
-add_loop_eager,                compile_time_instruction_count, 3004749893,  0.015
```

Comment:
this is the first time the auto generate file is used so thats why all numbers are changed since they are trimmed also.